<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="duankj-book" ignored="0" name="Testng_learn" tests="12" failures="1" timestamp="24 二月 2020 04:24:58 GMT" time="88.701" errors="0">
  <testcase name="f" time="4.675" classname="com.testing.runner.dataDriven"/>
  <testcase name="f" time="1.446" classname="com.testing.runner.dataDriven"/>
  <testcase name="f" time="1.16" classname="com.testing.runner.dataDriven"/>
  <testcase name="f" time="5.003" classname="com.testing.runner.dataDriven"/>
  <testcase name="f" time="1.095" classname="com.testing.runner.dataDriven"/>
  <testcase name="f" time="3.504" classname="com.testing.runner.dataDriven2"/>
  <testcase name="f" time="1.411" classname="com.testing.runner.dataDriven2"/>
  <testcase name="f" time="21.398" classname="com.testing.runner.dataDriven2">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="21.331" classname="com.testing.runner.dataDriven2">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="21.302" classname="com.testing.runner.dataDriven2">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.testing.runner.dataDriven2.f(dataDriven2.java:43)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="5.003" classname="com.testing.runner.dataDriven2"/>
  <testcase name="f" time="1.109" classname="com.testing.runner.dataDriven2"/>
</testsuite> <!-- Testng_learn -->
